### PR TITLE
Adding T2-Ixia topology file.

### PR DESCRIPTION
**Pull request by rraghav-cisco**
### Description of PR
As of now, we use T2 topology to deploy-minigraph in the T2 Ixia LC and then remove all the portchannels and other configs, to prepare it for ixia/snappi scripts. To make it more streamlined, we are adding a new topology, that uses only 4 connections in the testbed.

Adding a new topology file for T2-Ixia Cisco testbeds. This is only to provide support for deploy-mg. The new topology doesn't come with any Arista config files, since it is only for Ixia-Cisco connections.

Summary:
See the description.

### Type of change
- [ ] Bug fix
- [X] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [] 202205
- [X] 202305
- [ ] 202311

#### What is the motivation for this PR?
To avoid additional steps involved in bringing up the T2-ixia testbeds.

#### How did you do it?
Added a new topology.

#### How did you verify/test it?
Ran it on my testbed, deploy-mg is successful. 
```
sonic@sonic-ucs-m4-3:~/rraghav/Ixia/sonic-test/sonic-mgmt/ansible$ cat testbed.csv
# conf-name,group-name,topo,ptf_image_name,ptf,ptf_ip,ptf_ipv6,server,vm_base,dut,inv_name,auto_recover,comment
yy39-t2-ixia,t2-yy39,t2-ixia-cisco,docker-ptf-snappi,yy39top-ptf,172.27.147.46,,server_1,VM0100,yy39top-lc4,lab,False,SFd-at-YY39-Top
sonic@sonic-ucs-m4-3:~/rraghav/Ixia/sonic-test/sonic-mgmt/ansible$ 
```

Able to run delpoy-mg:
`sonic@snappi-sonic-mgmt:/data/ansible$ ./testbed-cli.sh deploy-mg yy39-t2-ixia ./lab ./password.txt -vvv`

```
PLAY RECAP ***********************************************************************************************************************************************************************
yy39top-lc4                : ok=67   changed=16   unreachable=0    failed=0    skipped=50   rescued=0    ignored=1   
```

Able to run snappi tests after that. 
```
===================================================================================== PASSES =====================================================================================
______________________________________________ test_global_pause[linecard_configuration_set0-chassis_single_line_card_single_asic] _______________________________________________
--------------------------------------------- generated xml file: /run_logs/ixia/8587/2024-01-27-08-28-11/tr_2024-01-27-08-28-11.xml ---------------------------------------------
INFO:root:Can not get Allure report URL. Please check logs
----------------------------------------------------------------------------- live log sessionfinish -----------------------------------------------------------------------------
08:37:39 __init__.pytest_terminal_summary         L0064 INFO   | Can not get Allure report URL. Please check logs
============================================================================ short test summary info =============================================================================
PASSED snappi_tests/multidut/pfc/test_multidut_global_pause_with_snappi.py::test_global_pause[linecard_configuration_set0-chassis_single_line_card_single_asic]
=================================================================== 1 passed, 3 warnings in 566.59s (0:09:26) ====================================================================
sonic@snappi-sonic-mgmt:/data/tests$ 
```

#### Any platform specific information?
For Cisco-8000 platforms only.

### CODE REVIEW

File: ansible/vars/topo_t2-ixia-cisco.yml
Comment (sdszhang) [inline ansible/vars/topo_t2-ixia-cisco.yml:1]: 't2-ixia-cisco' is vendor specific name.  Please make the file name more generic.

Comment (rraghav-cisco) [reply ansible/vars/topo_t2-ixia-cisco.yml:1]: @sdszhang : Renamed to t2-ixia-single-dut-masic.

**Comment by arlakshm** [comment]
@rraghav-cisco, can you please check if this PR is still needed.

**Comment by rraghav-cisco** [comment]
Its useful to anyone who wants to use a single LC - Ixia TB. But there is another PR which will use 3 DUTs, so closing this.

**Pull request closed without merging**